move_data_tags2.rb

```diff
@@ -0,0 +1,24 @@
+class MigrateTagsData < ActiveRecord::Migration
+
+  class OldTag < ActiveRecord::Base
+  end
+
+  class OldTagsTodoItems < ActiveRecord::Base
+  end
+
+  ActiveRecord::Base.transaction do
+
+    OldTagsTodoItems.all.each do |relation|
```

Comment:
Prawie dobrze :) Proponowane zmiany:

- Dodaj `belongs_to` do `OldTagsTodoItems` - będziesz mogła się bez problemu do niego odwoływać
- Nie potrzebujesz wyciągać `item` bo już teraz wiesz że zawsze będzie to `TodoItem`
- `old_tag` możesz wyciągnąć przez `relation.old_tag`

Poza tym wygląda dobrze :)